lib/ui/card.tsx

```diff
@@ -36,32 +47,73 @@ const CardTitle = React.forwardRef<
 >(({ className, ...props }, ref) => (
   <h3
     ref={ref}
-    className={cn(
-      "text-2xl font-semibold leading-none tracking-tight",
-      className
-    )}
+    className={cn("text-base font-semibold", className)}
     {...props}
   />
 ))
 CardTitle.displayName = "CardTitle"
 
-const CardDescription = React.forwardRef<
+const CardSubtitle = React.forwardRef<
   HTMLParagraphElement,
-  React.HTMLAttributes<HTMLParagraphElement>
+  React.HTMLAttributes<HTMLHeadingElement>
 >(({ className, ...props }, ref) => (
-  <p
+  <h3
     ref={ref}
-    className={cn("text-sm text-muted-foreground", className)}
+    className={cn("text-base font-normal text-muted-foreground", className)}
     {...props}
   />
 ))
-CardDescription.displayName = "CardDescription"
+CardSubtitle.displayName = "CardSubtitle"
+
+const CardInfo = React.forwardRef<
+  HTMLDivElement,
+  React.HTMLAttributes<HTMLDivElement>
+>(({ className, ...props }, ref) => (
+  <div
+    ref={ref}
+    className={cn("-ml-1 flex h-6 w-6 items-center justify-center", className)}
+  >
+    <TooltipProvider>
+      <Tooltip>
+        <TooltipTrigger
+          className="h-5 w-5 cursor-help text-muted-foreground"
+          title="Info"
+        >
+          <Icon icon={InfoCircleLine} size="md" />
+        </TooltipTrigger>
+        <TooltipContent>
+          <p {...props} />
+        </TooltipContent>
+      </Tooltip>
+    </TooltipProvider>
+  </div>
+))
+CardInfo.displayName = "CardInfo"
+
+const CardLink = React.forwardRef<
+  HTMLAnchorElement,
+  React.ComponentPropsWithoutRef<"a">
+>(({ className, ...props }) => {
+  return (
+    <a
+      className={cn(
+        "h-6 w-6 text-muted-foreground transition-colors hover:text-foreground",
+        className
+      )}
+      aria-label="Link"
```

Comment:
This is a magic string and it's not translated. We should use something that we receive as a prop. That way, we can even smoothly transition to using a link with text instead in case we wanted in the future.

Reply:
A way to do this is to receive the link as a parameter like this:

```
link?: {
  title: string,
  url: string
}
```

This way you can check if `link` is there, and then, in case it exists, it's forced to have `title` and `url`.